src/couch_views/README.md

```diff
@@ -7,42 +7,28 @@ Currently only map indexes are supported and it will always return the full inde
 Code layout:
 
 * `couch_views` - Main entry point to query a view
-* `couch_views_reader` - Reads from the index for queries
+* `couch_views_batch` - Dynamically determine optimal batch sizes for view indexers.
```

Comment:
Can you add in `couch_views_batch_impl` here.

Reply:
Will do, good catch.

Reply:
Done.